src/main/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtil.java

```diff
@@ -417,4 +417,38 @@ else if (sibling.getType() == TokenTypes.SINGLE_LINE_COMMENT) {
                 || BlockCommentPosition.isOnPackage(blockComment));
     }
 
+    /**
+     * Returns inline tag Node inside html tags from an HTML tag.
+     *
+     * @param ast html tag node.
+     * @return DetailNode of inline tag.
+     */
+    public static DetailNode getInlineTagNodeWithinHtmlElement(DetailNode ast) {
```

Comment:
please move this method back to Check, this update damage design and leak to Util very immature implementation. Force you to write weird UTs, .... .

we can always suppress violations when required and reasonable.
please restore method and add this SummaryJavadocCheck.java in suppression for now.
We will think on what to do with it at the end.

add suppression for amount of methods to https://github.com/checkstyle/checkstyle/blob/master/config/suppressions.xml

Reply:
Done.